irods/message/__init__.py

```diff
@@ -67,7 +67,7 @@ def __repr__(self):
     XML_Parser_Type.__members__ = {k:v for k,v in XML_Parser_Type.__dict__.items()
                                    if isinstance(v,XML_Parser_Type)}
 
-    PARSER_TYPE_STRINGS = {v:k for k,v in XML_Parser_Type.__members__.items() if v.value != 0}
+PARSER_TYPE_STRINGS = {v:k for k,v in XML_Parser_Type.__members__.items() if v.value != 0}
```

Comment:
A comment explaining the result of this expression would be very helpful.

Reply:
done.